ospec/tests/test-ospec.js

```diff
@@ -178,104 +93,178 @@ o.spec("ospec", function() {
 			o(output).deepEquals({tag: "div", children: children})
 		})
 	})
-	o.spec("async callback", function() {
-		var a = 0, b = 0
 
-		o.before(function(done) {
-			callAsync(function() {
+	o.spec(".only()", function () {
```

Comment:
Now that we can inject the reporter in `o.run` this could become a proper test in the main suite.

Reply:
How would that work?

Reply:
Like the reporter suite.

Reply:
Oh of course - yes.